### PR TITLE
Added ENV variables template and steps to set it up in README

### PR DESCRIPTION
**Pull request by BIJJUDAMA**
@Ashrockzzz2003 Pls review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,13 @@ This is the backend server for a web application for tracking the placement acti
     ```sh
     npm start
     ```
-
+### Create a .env file in the root directory of the project, and add the following environment variables:
+ ```sh
```
Comment (Ashrockzzz2003) [inline README.md:31]:
```suggestion
 ```env
```

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]
This should be step 2,
`npm start` is step 3.

Refer https://github.com/anokha-24/anokha_backend for an idea of how this should be presented in `README`.

File: README.md (hunk 1 above)
Comment (BIJJUDAMA) [reply README.md:31]: Got it

Comment (BIJJUDAMA) [reply README.md]: @Ashrockzzz2003 Please check now

Comment (BIJJUDAMA) [reply README.md]: Should I add some extra stuff like a guide??

Comment (Ashrockzzz2003) [reply README.md]: Change it to `env` the code block language.

File: README.md [hunk 2]
```diff
@@ -22,8 +22,15 @@ This is the backend server for a web application for tracking the placement acti
     ```sh
     npm install
     ```
+2. Create a .env file in the root directory of the project, and add the following environment variables:
+      ```env
```
Comment (Ashrockzzz2003) [inline README.md:26]: the alignment of code is bad. Do it patiently @BIJJUDAMA! 😄

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]
Format the `md` properly

File: README.md (hunk 2 above)
Comment (BIJJUDAMA) [reply README.md:26]: Oooh ok 
Thanks for the guidance

Comment (BIJJUDAMA) [reply README.md]: @Ashrockzzz2003 How about now?

**Review by Ashrockzzz2003 — APPROVED** [review]
LGTM

**Pull request merged**